scripts/hud/cgaz.js

```diff
@@ -311,6 +298,23 @@ class Cgaz {
 			}
 		}
 
+		if (this.snapAngles.length > this.snapZones?.length) {
+			const start = this.snapZones?.length;
+			const end = this.snapAngles.length;
+			for (let i = start; i < end; ++i) {
+				$.CreatePanel('Panel', this.snapContainer, `SnapZone${i}`);
+				this.snapZones.push(initZonePanel($(`#SnapZone${i}`)));
```

Comment:
yep, it does. I think I did it this way here to break up an otherwise hard to read line.

It's really not that bad though, I'll combine these lines.

Reply:
You don't need to combine the two lines, can just assign `const panel = $.CreatePanel` and then use `panel`.